kubernetes/resource_kubernetes_tokenrequest.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package kubernetes
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	apiv1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func resourceKubernetesTokenRequestV1() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceKubernetesTokenRequestCreateV1,
+		ReadContext:   resourceKubernetesTokenRequestReadV1,
+		UpdateContext: resourceKubernetesTokenRequestUpdateV1,
+		DeleteContext: resourceKubernetesTokenDeleteV1,
+
+		Schema: map[string]*schema.Schema{
+			"metadata": namespacedMetadataSchema("token request", true),
+			"spec": {
```

Comment:
This needs to be marked as computed because if we don't specify it then the default values are going to get filled in.